static/app/views/issueList/overview.tsx

```diff
@@ -289,8 +289,8 @@ class IssueListOverview extends Component<Props, State> {
 
     const {query} = location.query;
 
-    if (query !== undefined) {
-      return query as string;
+    if (typeof query === 'string') {
+      return query;
     }
```

Comment:
Another solution might be to call `decodeScalar` which will return the value, or first item in the array.

https://github.com/getsentry/sentry/blob/master/static/app/utils/queryString.tsx#L58-L72

Both values will already be strings because they're coming from `location.query.query`.

Reply:
Ooh thanks for pointing that out, didn't know it existed! Do agree that would be best